main/src/library/List.flix

```diff
@@ -749,7 +749,7 @@ namespace List {
 
     ///
     /// Returns the association list `xs` as a map.
-    /// If `xs` contains multiple values for the same key, the value appearing latest in `xs` is used.
+    /// If `xs` contains multiple values for the same key, the value appearing earliest in `xs` is used.
```

Comment:
Why? I would expect that if I have map and I insert the same key twice, the latter would take precedent?